config-windows.md

```diff
@@ -94,3 +94,23 @@ The following parameters can be specified:
         }
    }
 ```
+
+## <a name="configWindowsHyperV" />HyperV
+
+`hyperv` is an OPTIONAL field of the Windows configuration. If present, the container will be run with Hyper-V isolation. If omitted, the container will be run as a Windows Server container.
+
+The following parameters can be specified:
+
+* **`utilityvmpath`** *(string, OPTIONAL)* - specifies the path to the image used for the utility VM. If not supplied, the runtime will search the container filesystem layers from the bottom-most layer upwards, until it locates "UtilityVM", and default to that path.
+
+* **`sandboxpath`** *(string)* - specifies the root of the path to the sandbox to be used for the container.
```

Comment:
> … but I'm not entirely sure how I would represent that differently or how you suggest I alter this.

I suggest you use:

```
* **`sandboxpath`** *(string, REQUIRED)* - specifies the root of the path to the sandbox to be used for the container.
```

The relevant section from the [link][1] is:

> If an OPTIONAL parent property is an object with a REQUIRED child, the child is only REQUIRED if the parent is set. For example…

and the following example.  For an example of this in the current spec, see [`mounts[].destination`][2], where `mounts` is OPTIONAL, but any entry in that array MUST include `destination`.

[1]: https://github.com/wking/opencontainer-runtime-spec/blob/05da00bc76e1746276e977ab45957ab39c31c73a/spec.md#property-specification
[2]: https://github.com/opencontainers/runtime-spec/blame/559acdf05cd4dd0b57d720a3981147597d249280/config.md#L55